shim/dist/rpm/shim-rune.spec

```diff
@@ -13,8 +13,8 @@ Release: %{centos_base_release}%{?dist}
 Summary: shim for Inclavare Containers(runE)
 Group: Development/Tools
 License: Apache License 2.0
-URL: https://github.com/alibaba/%{PROJECT}
-Source0: https://github.com/alibaba/%{PROJECT}/archive/v%{version}.tar.gz
+URL: https://github.com%{PROJECT}
+Source0: https://github.com%{PROJECT}/archive/v%{version}.tar.gz
 
```

Comment:
`https://github.com/inclavare-containers` is not a URL for our program.
`https://github.com/inclavare-containers/archive/v%{version}tar.gz` is not the real package url.